core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -58,7 +58,11 @@ import scala.util.{Failure, Success, Try}
  *            enable.auto.commit=false
  *       3. Mirror Maker Setting:
  *            abort.on.send.failure=true
+ *
+ * @deprecated The original Mirror Maker is deprecated since release 3.0. Similar functionality can be
+ *    found in the Connect-based re-implementation by the same name (aka MM2).
```

Comment:
1. `The original Mirror Maker` -> could we say `The old Mirror Maker` ?
2. `Similar functionality can be found in the Connect-based re-implementation by the same name (aka MM2)` -> Could we change to: `Please use Connect-based re-implementation  in org.apache.kafka.connect.mirror.MirrorMaker (aka MM2) instead.`?

Reply:
Looking at other uses of the @deprecated javadoc tags, I see comments like "Since 2.4, use the PartitionReassignment Kafka API instead", which isn't far off from your suggestion. I'll reword, thanks.